scarb/src/resolver/mod.rs

```diff
@@ -247,8 +257,16 @@ mod tests {
     }
 
     fn resolve(
+        registry: MockRegistry,
+        roots: Vec<(&[ManifestDependency], PackageId)>,
+    ) -> anyhow::Result<Resolve> {
+        resolve_with_lock(registry, roots, &[])
+    }
```

Comment:
dead code?

Reply:
It's actually used exactly once, here: https://github.com/software-mansion/scarb/blob/7a5356a51fd4090f212f62a49b2c0b44c22d37dc/scarb/src/resolver/mod.rs#L616C2-L616C2